src/main/java/org/prebid/server/bidder/nanointeractive/NanointeractiveBidder.java

```diff
@@ -109,6 +103,15 @@ private ExtImpNanointeractive parseImpExt(Imp imp) {
         }
     }
 
+    private static Site modifySite(String reference, BidRequest request) {
```

Comment:
I'm sorry, I suggested `updateSite` but not I think that it will be more appropriate to call modifi**ed** site.
And pass only Site to the arguments, not all BidRequest